### DISCUSSION TITLE
Variant group or

### DISCUSSION THREAD

**Discussion by agufagit** [unocss/unocss#1421] [Ideas]
I'm finding myself applying same set of styles to two variants which are not chainable

```html
<input class="peer" />
<label class="peer-focus-within:( ... ) [&:not(peer-placeholder-shown)]:( ... )">test</label>
```

I think this `or` syntax would work
```
{a||b}:(c d) => a:c b:c a:d b:d
```

I can contribute if you want this feature

**Comment by antfu** [comment]
I feel that would be a bit too much. Maybe you could fork the variant group transformer inlined in your config to achieve that.

**Reply by louiss0** [reply]
I think it's a good idea to just be able to chain multiple variants. hover and focus are ones that are used the most please ignore the callback function and focus on the `|` syntax  being able to do 
```html 
<div class="hover|focus:(bg-red-500 text-gray-900)" > </div>

```

```css
:hover,
:focus {
background: rgb(255,255,59)
color: #lpepe
} 
```

Would be amazing I hope this feature does get added my life would be soo much easier
